sklearn/preprocessing/label.py

```diff
@@ -304,6 +304,26 @@ def fit(self, y):
         self.classes_ = unique_labels(y)
         return self
 
+    def fit_transform(self, y):
+        """Fit label binarizer and transform multi-class labels to binary labels.
+
+        The output of transform is sometimes referred to by some authors as the
+        1-of-K coding scheme.
+
+        Parameters
+        ----------
+        y : numpy array or sparse matrix of shape (n_samples,) or
+            (n_samples, n_classes) Target values. The 2-d matrix should only
```

Comment:
the first line is treated specially. You've let it wrap onto the second. I think the first line can be continued, but not like this. Try rendering the docs and seeing how it looks